assets/common/common.go

```diff
@@ -51,32 +54,35 @@ func HandleFatalError(err error, msg string) {
 	os.Exit(1)
 }
 
+// HandleNonFatalError ... Prints error to stderr if error is not nil, does nothing otherwise
 func HandleNonFatalError(err error, msg string) {
 	if err == nil {
 		return
 	}
 	fmt.Fprintln(os.Stderr, fmt.Sprintf("%s: %s", msg, err.Error()))
 }
 
-func ValidateAndBuildPostBody(input ConcourseInput) (method string, values url.Values, err error, emtpy bool) {
+// ValidateAndBuildPostBody ... Directs requests to appropriate delegates depending on if we deal with a file upload or a chat message
+func ValidateAndBuildPostBody(input ConcourseInput) (method string, values url.Values, emtpy bool, err error) {
 	var exists = false
 	if method, exists = slackMethodPaths[input.Source.Method]; exists == false {
-		return "", nil, errors.New(fmt.Sprintf("Method '%s' does not exist", input.Source.Method)), false
+		return "", nil, false, fmt.Errorf("Method '%s' does not exist", input.Source.Method)
```

Comment:
curious, why the change in return order?

Reply:
It was suggested by `golint` to specify the error as the last return parameter.